src/components/QueueEdit/QueueEdit.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { connect } from 'react-redux';
+import {
+  fetchStreamMarkerCreate,
+  fetchStreamMarkerList,
+} from '../TrackMarker/network';
+import {
+  fetchStreamQueueIntervalCreate,
+} from '../QueueInterval/network';
+import TrackMarker from '../TrackMarker/TrackMarker'
+import QueueInterval from '../QueueInterval/QueueInterval'
+
+
+function QueueEdit(props) {
+
+  const queue = props.data,
+        queueUuid = queue.uuid,
+        parentQueueUuid = queue.parentUuid,
+        trackMarkerMap = props.trackMarkerMap;
+
+  const markers = trackMarkerMap[queueUuid] || [];
+
+  const [formMarkerTimestamp, setFormMarkerTimestamp] = useState('');
+  const [lowerBoundMarkerUuid, setLowerBoundMarkerUuid] = useState('null');
+  const [upperBoundMarkerUuid, setUpperBoundMarkerUuid] = useState('null');
+
+  useEffect((queue, queueUuid) => {
+    async function loadData() {
+      await fetchStreamMarkerList(queue.track.uuid, queueUuid);
```

Comment:
Redux reducer is called inside this function. I'm not sure if this is a good pattern - should it be called inside of the component instead?